aviary/validation_cases/benchmark_tests/test_bench_FwFm.py

```diff
@@ -340,32 +328,32 @@ def bench_test_swap_4_FwFm_simple(self):
                     "final_mach": (0.3, "unitless"),
                     "mach_bounds": ((0.2, 0.8), "unitless"),
                     "initial_altitude": (35000.0, "ft"),
-                    "final_altitude": (500.0, "ft"),
+                    "final_altitude": (35.0, "ft"),
                     "altitude_bounds": ((0.0, 35000.0), "ft"),
                     "throttle_enforcement": "path_constraint",
                     "fix_initial": False,
                     "constrain_final": True,
                     "fix_duration": False,
-                    "initial_bounds": ((120.5, 361.5), "min"),
-                    "duration_bounds": ((5.0, 60.0), "min"),
-                    "no_climb": False
+                    "initial_bounds": ((120., 800.), "min"),
+                    "duration_bounds": ((5.0, 35.0), "min"),
+                    "no_climb": True,
                 },
-                "initial_guesses": {"times": ([241, 58], "min")},
+                "initial_guesses": {"times": ([241, 30], "min")},
             },
             "post_mission": {
                 "include_landing": True,
                 "constrain_range": True,
-                "target_range": (3360.0, "nmi"),
+                "target_range": (3368.0, "nmi"),
             },
         }
 
         prob = run_aviary(
-            'models/test_aircraft/aircraft_for_bench_FwFm_simple.csv', phase_info)
+            'models/test_aircraft/aircraft_for_bench_FwFm.csv', phase_info, max_iter=50)
```

Comment:
Are we hitting maxiter on this benchmark? It is the FLOPS/FLOPS test, so I would expect it to converge more quickly.

Reply:
Not hitting max_iter; I added it during debugging